app/components/post_list/post/body/add_members/add_members.tsx

```diff
@@ -41,22 +75,21 @@ const AddMembers = ({channelType, currentUser, location, post, theme}: AddMember
     const styles = getStyleSheet(theme);
     const textStyles = getMarkdownTextStyles(theme);
     const serverUrl = useServerUrl();
-    const postId: string = post.props.add_channel_member?.post_id;
-    const noGroupsUsernames = post.props.add_channel_member?.not_in_groups_usernames;
-    let userIds: string[] = post.props.add_channel_member?.not_in_channel_user_ids;
-    let usernames: string[] = post.props.add_channel_member?.not_in_channel_usernames;
+    let postId: string = '';
+    let noGroupsUsernames: string[] = [];
+    let userIds: string[] = [];
+    let usernames: string[] = [];
+    if (isAddMemberProps(post.props?.add_channel_member)) {
```

Comment:
0/5 but we could probably return early if this check is false since it looks like it's guaranteed to return early below because `postId` will be empty if this check is false